services/ui-src/src/components/pages/Dashboard/DashboardTable.tsx

```diff
@@ -129,7 +130,8 @@ interface DashboardTableProps {
 const getReportName = (report: ReportMetadataShape) => {
   const reportName = report.submissionName;
   const period = calculatePeriod(convertDateUtcToEt(report.createdAt));
-  return `${report.state} ${reportName} ${year} - Period ${period}`;
+  const fullStateName = States[report.state as keyof typeof States];
+  return `${fullStateName} ${reportName} ${year} - Period ${period}`;
```

Comment:
Question: The `year` variable here points to the current year. Would we want to pull the year from the report data somehow, so that older reports aren't mislabeled?

Reply:
Good question! I pinged Natalie to check if we will ever display older reports (reports from the previous year). Just in case we do, I'll refactor this to pull the year from the `report.createdAt` date 🙌